app/src/main/java/org/ole/planet/myplanet/ui/news/AdapterNews.java

```diff
@@ -113,13 +115,21 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
                 showShareButton(holder, news);
                 ((ViewHolderNews) holder).tvMessage.setText(news.getMessageWithoutMarkdown());
                 ((ViewHolderNews) holder).tvDate.setText(TimeUtils.formatDate(news.getTime()));
-                ((ViewHolderNews) holder).imgDelete.setOnClickListener(view -> new AlertDialog.Builder(context).setMessage(R.string.delete_record).setPositiveButton(R.string.ok, (dialogInterface, i) -> deletePost(news)).setNegativeButton(R.string.cancel, null).show());
-                ((ViewHolderNews) holder).imgEdit.setOnClickListener(view -> showEditAlert(news.getId(), true));
+                if(Objects.equals(news.getUserId(), currentUser.get_id())){
```

Comment:
what about admins and teamleaders?

Reply:
admins are not able to log into myPlanet. all other users (normal users/teamleaders) are the ones able to login and as in planet everyone can only delete what they created

Reply:
good about admins
but for teamleaders we then have to change stuff also in planet

Reply:
this is noted